.storybook/main.ts

```diff
@@ -1,4 +1,4 @@
-import { DocsOptions, Preset, StorybookConfig } from '@storybook/types';
+import { DocsOptions, Preset, StorybookConfig } from 'storybook/internal/types';
```

Comment:
Storybook's migration script made these changes. They consolidated to one package.